src/FirmwarePlugin/FirmwarePlugin.h

```diff
@@ -307,10 +307,6 @@ class FirmwarePlugin : public QObject
     /// @return A list of QUrl with the indicators
     virtual const QVariantList& modeIndicators(const Vehicle* vehicle);
 
-    /// Returns a list of CameraMetaData objects for available cameras on the vehicle.
-    /// TODO: This should go into QGCCameraManager
-    virtual const QVariantList& cameraList(const Vehicle* vehicle);
```

Comment:
So how does a custom build override the list of available cameras now?

Reply:
True, I will re-enable that ability. Personally I think it would be better to read some kinda resource file for this kinda stuff, like a json or whatever. But lemme think about it some first. I do think it's at least nice to have all the camera code together as long as we keep the same functionality.

Reply:
"Personally I think it would be better to read some kinda resource file for this kinda stuff, like a json or whatever."

I agree. In general for custom build tweaking I've been moving away from C++ code when possible to resource based stuff which can be overridden. Json for this certainly makes sense.

Reply:
So now they can override the json file in the resource system